services/src/main/scala/cromwell/pipeline/service/ProjectSearchFilterCleaner.scala

```diff
@@ -0,0 +1,33 @@
+package cromwell.pipeline.service
+
+import akka.actor.ActorSystem
+import cromwell.pipeline.utils.MiscConfig
+
+import java.time.Instant
+import scala.concurrent.ExecutionContext
+
+trait ProjectSearchFilterCleaner {
+
+  def cleanExpiredFilters(): Unit
+
+}
+
+object ProjectSearchFilterCleaner {
+
+  def apply(projectSearchFilterService: ProjectSearchFilterService, miscConfig: MiscConfig, system: ActorSystem)(
+    implicit executionContext: ExecutionContext
+  ): ProjectSearchFilterCleaner =
+    new ProjectSearchFilterCleaner {
+
+      import miscConfig.filtersDuration._
+
+      override def cleanExpiredFilters(): Unit = {
+        val _ = system.scheduler.scheduleAtFixedRate(cleanInterval, cleanInterval, getFiltersCleanRun, executionContext)
```

Comment:
I don't see why you would need this `val` definition if it's an only line
```suggestion
        system.scheduler.scheduleAtFixedRate(cleanInterval, cleanInterval, getFiltersCleanRun, executionContext)
```

Reply:
fixed